BedrockServer.cpp

```diff
@@ -1934,6 +1935,20 @@ void BedrockServer::_control(unique_ptr<BedrockCommand>& command) {
             __quiesceThread = nullptr;
             response.methodLine = "200 Unblocked";
         }
+    } else if (SIEquals(command->request.methodLine, "SetMaxPeerFallBehind")) {
+        // Look up the existing value so we can report what it was.
+        uint64_t existingValue = SQLiteNode::MAX_PEER_FALL_BEHIND;
+        response["previousValue"] = to_string(existingValue);
+
+        uint64_t newValue = command->request.calcU64("value");
+        if (newValue < 50) {
```

Comment:
any reason to allow 0-49 here vs just explicitly having it be 
```cpp
if (newValue == 0) {
    ...
}
```

Then maybe we have a set minimum limit of `50 or `100` commits that we also check and prevent it from getting set below that? I assume the reasoning for having _some_ limit is that it would force too much check pointing and potentially drive more thing into the blocking queue?

Reply:
The main reason is explained in this comment:
https://github.com/Expensify/Bedrock/pull/1577/files#diff-2d1c5f24a4b7ebc3076630d8ee3a1155ad5e7789abe72cff977e615541c319d6R65-R66

Reply:
If 10 is the magic number then why is this 50? Is it just some random multiple on 10?